src/47380.c

```diff
@@ -1,14 +1,114 @@
 #include "common.h"
+#include "ld_addrs.h"
+
+void func_8009D21C(s32 arg0, s32* arg1);
+
+typedef struct PhotoData {
+    u8 pad[0x3A0];
+} PhotoData; // Size: 0x3A0
 
-extern s32 D_800AC0F0;
 extern s32 D_800AE27C;
 extern s32 D_800AE280;
-extern s32 D_800B0598;
+extern PhotoData D_800B0598[60]; // Size: 0xD980 - All photos taken in a level
```

Comment:
Since you know the size of this it may be a good idea to add it to symbol_addrs to improve the disassembly and yeet some fake symbols

Reply:
oh, that's nice. didn't knew there was zero fake symbols